internal/tss/services/transaction_service.go

```diff
@@ -84,48 +84,41 @@ func (t *transactionService) NetworkPassphrase() string {
 }
 
 func (t *transactionService) BuildAndSignTransactionWithChannelAccount(ctx context.Context, operations []txnbuild.Operation, timeoutInSecs int64) (*txnbuild.Transaction, error) {
-	var tx *txnbuild.Transaction
-	var channelAccountPublicKey string
-	err := db.RunInTransaction(ctx, t.DB, nil, func(dbTx db.Transaction) error {
```

Comment:
I removed the call to `RunInTransaction` here because it was not being used. The dbTx was not being applied anywhere and considering there's only one database query, at the very last step here, then the database transaction serves no purpose here.